README.md

```diff
@@ -9,7 +9,7 @@ server.register({
     register: hhPlugin, 
     options: {
        // bootstrap with a prebuilt adapter
-        adapter: require('hapi-harvester/adapters/mongodb')({mongodbUrl: 'mongodb://localhost/test'})
+        adapter: require('hapi-harvester/adapters/mongodb')({mongodbUrl: 'mongodb://localhost/test', baseUri: server.info.uri})
```

Comment:
baseUrl would need to be a plugin property instead of adapter. Since we only want to do this work once and not rewrite it for every adapter that we create. See below.